src/components/data/EntitiesContent.js

```diff
@@ -346,9 +347,9 @@ const EntitiesContent = ({
         }, ['Download as TSV']),
         !snapshotName && h(MenuButton, {
           disabled: noEdit,
-          tooltip: noEdit ? 'You don\'t have permission to modify this workspace.' : 'Edit an attribute of the selected rows',
+          tooltip: noEdit ? 'You don\'t have permission to modify this workspace.' : 'Edit a field of the selected rows',
```

Comment:
So valuable to get user feedback like this. I wouldn't have thought that this wasn't obvious. 👍

Reply:
The terms `attribute` and `entity` have always confused me  🤷🏼  Might as well use `thing` in lieu of `entity` 😛